update.sh

```diff
@@ -6,8 +6,8 @@
 # Description:
 #   An update bash script for engelsystem
 
-# Add upstream 
+# Add upstream
 git remote add upstream http://www.github.com/fossasia/engelsystem.git
```

Comment:
Do you notice that "origin" is also "http://www.github.com/fossasia/engelsystem.git"?
No need to define a "remote" with the same URL as "origin".